pyphare/pyphare/pharein/mhd_model.py

```
@@ -19,8 +19,55 @@ def defaulter(self, input, value):
         if self.dim == 3:
             return lambda x, y, z: value
 
+    def st_defaulter(self, input, value):
```

Ruff (0.15.20)

[error] Function argument `input` is shadowing a Python builtin

(A002)